### PR TITLE
Body Pillows :^)

### PR DESCRIPTION
**Pull request by ShadowDeath6**
### Intent of your Pull Request

Adds a new subtype of box, body pillows. They work similarly(identically) to hugboxes, but are a lot cuter and fireproof-er.

https://gyazo.com/7beb98d643c0b13e9c69778ac70c60b7

Thanks to @Kmc2000 for the sprite.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -662,3 +662,27 @@
...
+	icon_state = "pillowbox"
+	item_state = "pillowbox"
+	burn_state = -1 //nobody can burn my love
+	attack_verb = list("squished", "fluffed", "slapped", "made a smug gesture toward", "bopped", "smacked", )
+	hitsound = "sound/weapons/thudswoosh.ogg"
+	foldable = null
+
+/obj/item/weapon/storage/box/pillow/attack_self(mob/user)
+	..()
+	var/msg = pick("Uguu~", "Kyaa!", "B-baka!", "Cute!", "Aww!", "Smug!", "Kawaii!", "Desu~", "N-not there!", "Nyaa~", "Fufu~", "Sugoi!")
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm:679]: NOPE

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -662,3 +662,27 @@
...
+	user.changeNext_move(CLICK_CD_MELEE)
+	playsound(loc, "sound/weapons/thudswoosh.ogg", 50, 1, -5)
+	user.visible_message("<span class='notice'>[user] squeezes \the [src].</span>","<span class='notice'>You squeeze \the [src]. [msg]</span>")
+	return
+
+/obj/item/weapon/storage/box/pillow/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is cuddling \ with the [src.name]! It looks like \he's trying to commit social suicide.</span>")
+	src.say("You make my kokoro go doki-doki, [user]!")
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm:687]: NOPE

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -662,3 +662,27 @@
...
 	new /obj/item/stack/medical/ointment(src)
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
+
+
+/obj/item/weapon/storage/box/pillow
+	name = "body pillow"
+	desc = "A pillow with some generic cartoon girl on the front. <i>Certain</i> people might like this."
+	icon_state = "pillowbox"
+	item_state = "pillowbox"
+	burn_state = -1 //nobody can burn my love
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm:672]: NOPE

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 4]
```diff
@@ -662,3 +662,27 @@
...
+	attack_verb = list("squished", "fluffed", "slapped", "made a smug gesture toward", "bopped", "smacked", )
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/boxes.dm:673]: NOPE

**Comment by ToGWtF** [comment]
We have awakened the Goof.

**Comment by ShadowDeath6** [comment]
Note: These won't spawn in maint, they're currently adminbus only. I think somebody is working on a maint loot update (cough cough) so hopefully it'll be addressed then.

**Comment by X-TheDark** [comment]
In before Goof ports (with modifications) this and shows us how it's REALLY done.

**Comment by Iamgoofball** [comment]
no instead I'm going to declare this a meme zone and post the entire bee movie script if this isn't closed :ok_hand:

**Comment by Iamgoofball** [comment]
[REDACTED]

bad meems

**Comment by ShadowDeath6** [comment]
👌 memed

Anyway, this is semi-serious and I'd like for people to give their honest opinion on what they think of it.

**Comment by Alek2ander** [comment]
as long as it permabans everyone who touches it

**Comment by Kmc2000** [comment]
ATTENTION

Level 69 viral biohazard detected on this station, all crew must contain the outbreak (of memes)

**Comment by oisin100** [comment]
What even... 
:+1:  when you can take the fluffing out of pillows and fill them with glass shards.

EDIT: Oh Nevermind. You haven't actually added pillows... Ill go cry in a corner now.

**Comment by TheSadFox** [comment]
Needs to have a variant with Horo, otherwise i will be forced to :-1:  this.

Actually make it a donator feature, and replace their internals box with a pillow with their waifu on it!

**Comment by Kmc2000** [comment]
@SeikoTheSeal what's a horo?

Also, a waifu request form? so i can sprite it?

(meme zone ahead)

**Comment by TheSadFox** [comment]
> Doesn't know who Horo is
> Bro what the fuck

https://cs.sankakucomplex.com/data/ff/1d/ff1d188139b66fafbac122035b6384d5.png?5078173

http://myanimelist.net/anime/2966/Ookami_to_Koushinryou

**Comment by ghost** [comment]
and we're done

**Comment by Notamaniac** [comment]
Which is good, since that sprite was heresy even to anime

**Comment by ShadowDeath6** [comment]
I'd prefer this stay up so other people who actually play the game can comment on it, as most people I've asked (granted most of the people who talk to me openly are people who agree with me) think it'd be a funny addition to the game.
As for the sprite, I can't do much about it, but I'm entirely open to modifications which make it viable for the players to use or admins to abuse, since it won't actually be normally available.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 5]
```diff
@@ -662,3 +662,27 @@
...
+/obj/item/weapon/storage/box/pillow/attack_self(mob/user)
+	..()
+	var/msg = pick("Uguu~", "Kyaa!", "B-baka!", "Cute!", "Aww!", "Smug!", "Kawaii!", "Desu~", "N-not there!", "Nyaa~", "Fufu~", "Sugoi!")
+	user.changeNext_move(CLICK_CD_MELEE)
+	playsound(loc, "sound/weapons/thudswoosh.ogg", 50, 1, -5)
+	user.visible_message("<span class='notice'>[user] squeezes \the [src].</span>","<span class='notice'>You squeeze \the [src]. [msg]</span>")
+	return
+
+/obj/item/weapon/storage/box/pillow/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is cuddling \ with the [src.name]! It looks like \he's trying to commit social suicide.</span>")
```
Comment (Cruix) [inline code/game/objects/items/weapons/storage/boxes.dm:686]: This is stolen from fanny packs.

**Comment by AsV9** [comment]
and that's why adam no longer has write.

**Comment by Super3222** [comment]
Welp. We can all remember that this PR happened at one point in history.

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 5 above)
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/boxes.dm:686]: Still accurate, though. I knew it was from something.

**Comment by oisin100** [comment]
No need to close it. As stated earlier its adminbus only. Although i do want to see actual pillows..

**Comment by Kmc2000** [comment]
@Notamaniac

Whats wrong with the sprite? 

Its not TERRIBLE

**Comment by Iamgoofball** [comment]
Did you guys fire Adam for closing a seriously cringey PR? Goddamn, I knew the council was retarded but I didn't know the brain cancer stage was that bad.

**Comment by AsV9** [comment]
no we just removed his write because he was pushing directly to dev.

**Comment by ghost** [comment]
Wait, I did what when

Was that what happened to that slime processor thing I was wondering why it didnt make a pr

**Comment by Iamgoofball** [comment]
You guys do realize there's a protected branch option on github that prevents this right

**Comment by DavidEdwards** [comment]
Our `master` branch is protected, but not the `dev` branch.

**Pull request closed without merging**